llama/Makefile

```diff
@@ -88,24 +103,27 @@ HIP_FLAGS := \
 	-Wno-pass-failed \
 	-Wno-deprecated-declarations \
 	-Wno-unused-result \
-	-Xclang \
-	--dependent-lib=msvcrt \
 	-I. \
 	$(foreach arch, $(HIP_ARCHS), --offload-arch=$(arch))
 
 ifeq ($(OS), Linux)
-	HIP_FLAGS += $(foreach arch, $(LINUX_HIP_ARCHS), --offload-arch=$(arch))
+	HIP_FLAGS += $(foreach arch, $(LINUX_HIP_ARCHS), --offload-arch=$(arch)) -fPIC
+	CUDA_FLAGS += -fPIC 
+	CFLAGS += -Xcompiler -fPIC -D_GNU_SOURCE
+	CXXFLAGS += -Xcompiler -fPIC -D_GNU_SOURCE
+else ifneq (,$(findstring MINGW,$(OS)))
+	HIP_FLAGS += -Xclang --dependent-lib=msvcrt
 endif
 
-RUNNERS := ollama_runner
+RUNNERS := ollama_runner$(EXE_EXT)
```

Comment:
Do we need `EXE_EXT`? I _think_ it will auto append .exe on windows

Reply:
At least one of them didn't in my testing.